packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -700,7 +690,8 @@ export default {
                 const data = {
                     basketId: this.basket.id,
                     customerNotes: {
-                        noteForRestaurant: this.userNote
+                        noteForRestaurant: this.userNotes.delivery || this.userNotes.restaurant,
```

Comment:
Does this still work if `this.userNotes`  doesn't exist?

Reply:
It should always exist as it's made in the initial creating of the module state. 🙂

Reply:
A question rather than a comment: what's `userNotes.restaurant`? It looks very similar to `userNotes.kitchen`.